buildconfig/CMake/Packaging/launch_mantidworkbench.sh.in

```diff
@@ -1,34 +1,23 @@
 #!/bin/sh
 #
-# Launch Mantidplot using any necessary LD_PRELOAD or software collection behaviour
```

Comment:
There's a fair few MantidPlot references still here. While we're editing can we get rid of them?